pkg/api/register.go

```diff
@@ -70,6 +69,7 @@ func init() {
 	)
 
 	// Register Unversioned types
+	Scheme.AddKnownTypes("", &unversioned.ListOptions{})
```

Comment:
Yes - this will work. Also, if we need to register ListOptions in "extensions", this will also work correctly.